exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -50,6 +50,22 @@
         "info": false
       }
     );} );
+
+    //Close the cancellation status popup
+    function refreshStatus() {
+      //Close PopUp Modal
+      $("#queryCancelModal").modal("hide");
+      location.reload(true);
+    }
+
+    //Cancel query & show cancellation status
+    function cancelQuery() {
+      document.getElementById("cancelTitle").innerHTML = location.hostname;
```

Comment:
Why this is needed?

Reply:
Indicating the host to which the cancelation was issued.